test/controllers/ExcludeJourneyControllerSpec.scala

```diff
@@ -178,21 +180,23 @@ class ExcludeJourneyControllerSpec extends TestUtils {
 
     "return a NoContent" when {
 
-      "the journey key is valid" in {
-        lazy val request = FakeRequest("POST", s"/income-tax-submission-service/income-tax/nino/AA000000A/sources/exclude-journey/$taxYear")
-          .withHeaders("mtditid" -> "1234567890", "sessionId" -> "sessionId")
-          .withJsonBody(Json.obj("journey" -> INTEREST))
+      allJourneys.foreach { key =>
```

Comment:
`controller.allJourneys.foreach` can be used instead and the `allJourneys` variable declaration in this file can be removed.